code/Items/Perks/Disguiser.cs

```diff
@@ -9,9 +9,9 @@ namespace TTT;
 public partial class Disguiser : Perk
 {
 	[Net, Local]
-	public bool IsEnabled { get; set; } = false;
+	public bool IsActive { get; set; } = false;
```

Comment:
There is an `Enabled` property on a component, I think this rename clears things up.